app/src/pages/inside/stepPage/modals/editDefectModals/makeDecisionModal/optionsStepForm/optionsStepForm.jsx

```diff
@@ -25,15 +25,35 @@ import { DefectTypeItemML } from 'pages/inside/common/defectTypeItemML';
 import { defectTypesSelector } from 'controllers/project';
 import { ScrollWrapper } from 'components/main/scrollWrapper';
 import { ExecutionInfo } from 'pages/inside/logsPage/defectEditor/executionInfo';
+import { Accordion } from 'pages/inside/common/accordion';
+import {
+  CURRENT_EXECUTION_ONLY,
+  SOURCE_DETAILS,
+} from 'pages/inside/stepPage/modals/editDefectModals/constants';
+import { InputRadioGroup } from 'components/inputs/inputRadioGroup';
+import { LogItem } from 'pages/inside/logsPage/defectEditor/logItem';
 import { messages } from './../../messages';
 import styles from './optionsStepForm.scss';
 
 const cx = classNames.bind(styles);
 
-export const OptionsStepForm = ({ info }) => {
+export const OptionsStepForm = ({ info, toggleAccordionTab, itemData }) => {
```

Comment:
If you want to reuse specific accordion tabs logic, it makes sense to move it to a custom hook.
F.e. `useAccordionTabsState` and use it wherever you need it.